packages/loader/container-loader/src/containerContext.ts

```diff
@@ -224,7 +224,7 @@ export class ContainerContext implements IContainerContext {
     public setConnectionState(connected: boolean, clientId?: string) {
         const runtime = this.runtime;
 
-        assert(connected === this.connected, 0x0de /* "Mismatch in connection state while setting" */);
+        // assert(connected === this.connected, 0x0de /* "Mismatch in connection state while setting" */);
```

Comment:
I think we will need to deprecate connected property or make it calculatable from setConnectionState() calls how it's done in ContainerRuntime. Later is interesting because it uses initial value from context.connected, but after that it simply calculates the value. I believe it should be fine to always start with 'false', because Container.setLoaded() (called on all transitions like boot and reload) should let runtime know about actual state.